SWEgit/src/com/company/Main.java

```diff
@@ -295,18 +295,8 @@ public static void getmin3nums(String[]arr)
 	    	System.out.println(TempArr[1]);
 	    	System.out.println(TempArr[2]);
 	}
-}
-
-
-package GetMax3Nums;
 
-/**
- *
- * @author Blancko
- */
-class GetMax3Nums 
-{ 
-    static void printlargest(int arr[], int arr_size) 
+   static void GetMax3Nums(int arr[], int arr_size) 
```

Comment:
the input will enter your function as string so take it string and convert it to int in you function ..... hint : see median function